src/lib/utils/props.ts

```diff
@@ -29,7 +29,7 @@ export function createStyles(
 interface CreateClassesOptions {
   component?: string;
   element?: string;
-  userClasses?: string;
+  userClasses?: string | null;
```

Comment:
Why?

Reply:
Because I had an error `type string | undefined | null is not assignable to type string | undefined`. `string | undefined | null` is the type of the html "class" attribute